server/schedule/plan/plan.go

```diff
@@ -83,3 +84,10 @@ func SetResource(resource interface{}) Option {
 		plan.SetResource(resource)
 	}
 }
+
+// SetResource is used to generate Resource for plan
+func SetResourceWithStep(resource interface{}, step int) Option {
```

Comment:
Where is it used?

Reply:
https://github.com/tikv/pd/pull/5544 will need it